docs/sources/flow/reference/components/beyla.ebpf.md

```diff
@@ -0,0 +1,270 @@
+
+Contains the following blocks:
+[services]: #services-block
+
+### services block
+
+In some scenarios, Beyla will instrument a big variety of services; for example, as a Kubernetes DaemonSet that instruments all the services in a node. This block allows you to filter the services to instrument based on their metadata.
+
+Name | Type | Description                                               | Default | Required
+---- | ---- |-----------------------------------------------------------| ------- | --------
+`name ` | `string` | The name of the service to match. | | no
+`namespace` | `string` | The namespace of the service to match. | | no
+`open_ports` | `string` | The port of the running service for Beyla automatically instrumented with eBPF. | | no
+`path` | `string` | The path of the running service for Beyla automatically instrumented with eBPF. | | no
+
+`name` defines a name for the matching instrumented service. It will be used to populate the `service.name` OTEL property and/or the `service_name` Prometheus property in the exported metrics/traces.
+`open_port` accepts a comma-separated list of ports (for example, `80,443`), and port ranges (for example, `8000-8999`). If the executable matching only one of the ports in the list, it is considered to match the selection criteria.
+`path` accepts a regular expression to be matched against the full executable command line, including the directory where the executable resides on the file system.
+
+
+### output block
+
+The `output` block configures a set of components to forward resulting telemetry data to.
+
+The following arguments are supported:
+
+Name      | Type                     | Description                           | Default | Required
+----------|--------------------------|---------------------------------------|---------|---------
+`traces`  | `list(otelcol.Consumer)` | List of consumers to send traces to.  | `[]`    | no
+
+You must specify the `output` block, but all its arguments are optional.
+By default, telemetry data is dropped.
+Configure the `traces` argument accordingly to send traces data to other components.
+
+## Exported fields
+
+The following fields are exported and can be referenced by other components.
+
+Name      | Type                | Description
+----------|---------------------|----------------------------------------------------------
+`targets` | `list(map(string))` | The targets that can be used to collect metrics of instrumented services with eBPF.
+
+For example, the `targets` can either be passed to a `discovery.relabel` component to rewrite the targets' label sets or to a `prometheus.scrape` component that collects the exposed metrics.
+
+The exported targets use the configured [in-memory traffic][] address specified by the [run command][].
+
+[in-memory traffic]: {{< relref "../../concepts/component_controller.md#in-memory-traffic" >}}
+[run command]: {{< relref "../cli/run.md" >}}
```

Comment:
```suggestion
[in-memory traffic]: ../../../concepts/component_controller#in-memory-traffic
[run command]: ../../cli/run/
```

We should not be using relref links in any new documentation. The relref short code is deprecated.